v3/src/components/case-table/column-header.tsx

```diff
@@ -44,7 +44,6 @@ export const ColumnHeader = ({ column }: Pick<THeaderRendererProps, "column">) =
       case "Enter":
       case "Tab":
         handleClose(true)
-        e.currentTarget.blur()
```

Comment:
I'm curious about this change, which seems somewhat unrelated. Happy to see it go, but do you have some deeper understanding that led to its removal?

Reply:
It was part of my experimentation with the Enter key that should have been removed when I added `e.stopPropagation()`